services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouterController.java

```diff
@@ -293,6 +293,43 @@ public RouteTableWebJson getOrCreateSubnetRouteTable(@PathVariable String projec
         return new RouteTableWebJson(routeTable);
     }
 
+    /**
+     * Create neutron subnet routeTable
+     * @param projectid
+     * @param subnetid
+     * @param resource
+     * @return
+     * @throws Exception
+     */
+    @RequestMapping(
+            method = POST,
+            value = {"/project/{projectid}/subnets/{subnetid}/neutron-routetable"})
```

Comment:
The api for creating subnet route table for Neutron should be like this:
```
/project/{projectid}/subnets/{subnetid}/routetable
```

Please refer to https://github.com/futurewei-cloud/alcor/blob/master/docs/modules/ROOT/pages/mgmt_services/route_manager.adoc#subnet_add_neutron_routetable

Reply:
Sure